### ISSUE TITLE
Improve Preview URL Handling

### ISSUE THREAD

**Issue by tobiasschweizer** [dhlab-basel/Sipi#41]
Improve creation of preview URL in `make_thumbnail.lua`

Do not use hard coded strings, but us settings in config file (Sipi domain):

`preview_path = "http://localhost:1024/thumbs/" .. tmpname .. "_THUMB.jpg" .. "/full/full/0/default.jpg",`

This means we need a constant in the config file representing the SIpi domain.

**Comment by loicjaouen** [comment]
I am not sure to understand you; do you wish to set the host name a config file or to extract it from the request's header (if possible, that would be smart)?

Looking at the code, I didn't see how to set the server's name in a config file or an argument for it in `Server`'s constructor call, so it will most probably remain `localhost`.

If it is to some interest to anybody: I worked around it by this apache proxy/substitute the following config allowing us to access sipi as `http://platec.unil.ch/sipi/` :

sipi.conf:

```
<Location "/sipi/">

  ProxyPass "http://localhost:1024/"
  ProxyPassReverse "http://localhost:1024/"

  AddOutputFilterByType SUBSTITUTE application/json
  Substitute "s|localhost:1024|platec.unil.ch/sipi|ni"
</Location>
```

**Comment by subotic** [comment]
resolved in #125
